datasets/wmt_mlqe_task1/README.md

```diff
@@ -0,0 +1,187 @@
+---
+annotations_creators:
+- expert-generated
+- machine-generated
+language_creators:
+- found
+languages:
+- de
+- en
+- et
+- ne
+- ro
+- ru
+- si
+- zh
```

Comment:
The languages here depend on the configuration, so it must be formatted by providing the set of languages per config. For example
```
languages:
  en-de:
  - en
  - de
  en-ru:
  - en
  - ru
```
(must be filled for each config)

Can you change that please ?

same for task2